### PR TITLE
Bump 0xProto to v2.00, add livecheck

### PR DESCRIPTION
**Pull request by arjpar**
Also updated sha256 value

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=pullrequests).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/font-0xproto.rb [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
   name "0xProto"
   desc "Programming font focused on source code legibility"
   homepage "https://github.com/0xType/0xProto"
-
+  
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
```
Comment (bevanjkay) [inline Casks/font-0xproto.rb:14]:
```suggestion
```

It doesn't look like the repository uses GitHub pre-releases, or has tags without releases, so the `livecheck` block shouldn't be necessary. The default `:git` strategy will be inferred automatically from the `url`.

File: Casks/font-0xproto.rb [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-  sha256 "100c3478d17f0bed536173944b81a72d989195dd7a3cb84f7d6bacc429bad211"
+  version "2.000"
+  sha256 "ef69ffaa0f7ebab546081aa9b551b34db12f832627b75cb420164a07d9c6b4d0"
 
   url "https://github.com/0xType/0xProto/releases/download/#{version}/0xProto_#{version.dots_to_underscores}.zip"
   name "0xProto"
   desc "Programming font focused on source code legibility"
   homepage "https://github.com/0xType/0xProto"
-
+
```
Comment (bevanjkay) [inline Casks/font-0xproto.rb:9]:
```suggestion

```

Comment (bevanjkay) [inline Casks/font-0xproto.rb:9]: @arjpar The spaces need to be removed here to pass `brew style font-0xproto`

**Review by bevanjkay — APPROVED** [review]

File: Casks/font-0xproto.rb (hunk 2 above)
Comment (arjpar) [reply Casks/font-0xproto.rb]: @bevanjkay Sounds good, added a commit which removes the trailing space.

**Comment by bevanjkay** [comment]
Thank you @arjpar 🎉

**Pull request merged**